### DISCUSSION TITLE
Article about Indexer & Dictionary requests (French)

### DISCUSSION THREAD

**Discussion by gege83var** [capsule-corp-ternoa/ternoa-subql#116] [General]
Hi everyone,

This is an introductory article about the Indexer and the Dictionary (What is it, what can be done with it and how)

https://medium.com/@gege83var/d%C3%A9couverte-de-lindexer-ternoa-1929c1b2846

Enjoy :D

**Comment by ipapandinas** [comment]
hi @gege83var - amazing article, you make it much easier to understand the Indexer! Thank you for this contribution, I pin it and share it with the rest of the team 🚀
